### ISSUE TITLE
Could we automatically set the baseUrl for Cypress?

### ISSUE THREAD

**Issue by chrisvfritz** [vuejs/vue-cli#847]
I might be missing something, but it looks like the only reason we need to set the `VUE_DEV_SERVER_URL` env var for Cypress is because we don't set the `baseUrl`. What would you think about replacing:

``` js
'--env', `VUE_DEV_SERVER_URL=${url}`,
```

with:

``` js
'--config', `baseUrl=${url}`,
```

in `@vue/cli-plugin-unit-jest/index.js`?

Then in Cypress tests, we wouldn't have to write:

``` js
cy.visit(Cypress.env('VUE_DEV_SERVER_URL'))
```

but could instead simplify to:

``` js
cy.visit('/')
```

**Comment by yyx990803** [comment]
Ah cool, wasn't aware of the `baseUrl` option. Feel free to open a PR.

**Comment by yyx990803** [comment]
Closed via #879
